### ISSUE TITLE
sort out tax reporting for Teams

### ISSUE THREAD

**Issue by whit537** [gratipay/gratipay.com#960]
Reticketed from #901.

> we need to record country and EIN for the team participant&mdash;this is the legal entity that is considered the contracting party in the relationship with those receiving funds. We're going to have to accept U.S. only companies to start with. However, I can't see us launching this without allowing for non-U.S. contractors. Gittip itself has people outside the U.S. that we need to deal in from the start. See [Publication 515 (2013), Withholding of Tax on Nonresident Aliens and Foreign Entities](http://www.irs.gov/publications/p515/ar01.html). It sounds like the short story is that non-resident aliens will need to get an ITIN (like an SSN; form [W-7](http://www.irs.gov/pub/irs-pdf/fw7.pdf)) and then will be taxed 30%(!) unless there's a treaty with their country (which there probably is? no idea, really). This begs the question: what about money we're moving right now? It puts rather a damper on Gittip to have to withhold 30% of everything paid out outside the U.S. Also a damper to have to require ITINs, etc. Hmm ...

> - need to require opt-in plus paperwork to receive money from a team
  - storing SSNs/ITINs ... what does that do to our hosting situation? Do we need to get into PCI compliance here?
  - again, begging the question of what this would then mean for our existing setup&mdash;would the IRS have us withhold 30% from all payouts?

**Comment by whit537** [comment]
I'm pretty sure that https://github.com/gratipay/inside.gratipay.com/issues/242 covers this. Adding to "Bring Back Payroll" so we notice to close this if we're able to.

**Comment by whit537** [comment]
Yeah, I closing this as a dupe of https://github.com/gratipay/inside.gratipay.com/issues/242.

**Comment by whit537** [comment]
tl;dr is that Teams are legal entities, responsible for:

 - the legal relationship between they and their members
 - tax reporting for money they allow their members to take
 - their own tax obligations
